onprc_ehr/test/src/org/labkey/test/tests/onprc_ehr/ONPRC_EHRTest2.java

```diff
@@ -967,6 +968,67 @@ public void testManageCases() throws Exception
         waitForElementToDisappear(Ext4Helper.Locators.window("Manage Cases: " + SUBJECTS[0]));
     }
 
+    @Test
+    public void testFormTriggers()
+    {
+        String aliveAnimal = "99998";
+        String deadAnimal = "99999"; //Mark an animal dead.
+
+        Map<String, Map<String, String>> formsToTest = Map.of(
+                "Birth", Map.of("allowDatesInDistantPast", "true", "allowFutureDates", "true", "allowDeadIds", "true", "allowAnyId", "true"),
```

Comment:
Can you add another property to the inner maps, gridToTest which will be the grid in the form to test.

Reply:
Done